covid_graphs/covid_graphs/scripts/country_dashboard.py

```diff
@@ -71,17 +71,19 @@ def create_dashboard(
                 if event != OTHER
             ],
             value=BK_20200411.name,
-            style={"width": "40%"},
+            style={"width": "220px", "margin": "4px 0px"},
```

Comment:
Are pixels the recommended unit even on very high definition displays? E.g. on a 4K?

And I actually prefer even higher margin, perhaps 6 pixels.

Reply:
`px` doesn't actually mean pixels, it means "device independent pixels". They're defined in terms of viewing angle and they're typically implemented as 1 physical pixel on medium resolution screens and 2 physical pixels on high resolution screens.

Reply:
TIL!